graphs/dijkstra.py

```diff
@@ -3,18 +3,18 @@ def __init__(self, vertices):
         self.V = vertices
         self.graph = [[0 for column in range(vertices)] for row in range(vertices)]
 
-    def printSolution(self, dist):
+    def printsolution(self, dist):
         print("Vertex \t Distance from Source")
         for node in range(self.V):
             print(node, "\t\t", dist[node])
 
-    def minDistance(self, dist, sptSet):
+    def mindistance(self, dist, sptSet):
```

Comment:
```suggestion
    def minimum_distance(self, distances, spt_set) -> int:
```
What is `spt_set` supposed to be?!?  There must be a better name.